doc/stdenv/cross-compilation.chapter.md

```diff
@@ -44,7 +44,11 @@ The exact schema these fields follow is a bit ill-defined due to a long and conv
 
 `config`
 
-: This is a 3- or 4- component shorthand for the platform. Examples of this would be `x86_64-unknown-linux-gnu` and `aarch64-apple-darwin14`. This is a standard format called the "LLVM target triple", as they are pioneered by LLVM. In the 4-part form, this corresponds to `[cpu]-[vendor]-[os]-[abi]`. This format is strictly more informative than the "Nix host double", as the previous format could analogously be termed. This needs a better name than `config`!
+: This is a 3-, 4-, or 5- component shorthand for the platform.  Examples of this would be `x86_64-unknown-linux-gnux32`, `aarch64-apple-darwin14`, and `mips64el-unknown-linux-muslabin32`.  This is a standard format [pioneered by autoconf](https://www.gnu.org/software/autoconf/manual/autoconf-2.65/html_node/System-Type.html#System-Type) and [adopted by LLVM](https://clang.llvm.org/docs/CrossCompilation.html#target-triple).
+
+In the 5-part form, this corresponds to `[cpu]-[vendor]-[os]-[libc][abi]`; note that there is no hyphen separating the `[libc]` field from the `[abi]` field. This format is strictly more informative than the "Nix system double", the format used by the nix interpreter.  This needs a better name than `config`!
+
+This field should be *canonicalized*.  The rules for canonicalizing a tuple are kept in the `config.sub` file in the source code for `gnu-config`.
```

Comment:
I don't understand what this means. Is "should be canonicalized" a recommendation to the user? What is `gnu-config`and where is this `config.sub`file? I suppose you mean [this file](https://github.com/gcc-mirror/gcc/blob/32a06ce38a38bf37db468f0e6c83520fcc221534/config.sub)?

Unless you want to clarify this, I'd just leave it out and fix later if the need arises.

```suggestion
```